zds/settings/dev.py

```diff
@@ -76,3 +76,4 @@
 }
 
 ZDS_APP['site']['url'] = 'http://127.0.0.1:8000'
+ZDS_APP['site']['dns'] = 'http://127.0.0.1:8000'
```

Comment:
```suggestion
ZDS_APP['site']['dns'] = '127.0.0.1:8000'
```

Si on veut avoir `<meta property="og:image:url" content="http://127.0.0.1:8000/static/images/favicons/512.png">` et non pas `<meta property="og:image:url" content="http://http://127.0.0.1:8000/static/images/favicons/512.png">`. C'est aussi ce qui est utilisé dans [`zds/settings/beta.py`](https://github.com/zestedesavoir/zds-site/blob/dev/zds/settings/beta.py) d'ailleurs.